resources/lambda/browsershot.js

```diff
@@ -1,6 +1,7 @@
 const fs = require('fs');
-const {execSync} = require('child_process');
+const { execSync } = require('child_process');
 const chromium = require('chrome-aws-lambda');
+const AWS = require('aws-sdk');
```

Comment:
How is `aws-sdk` pulled in? Is this just available when deploying the Lambda?

Reply:
I believe so. I just put that line there and it worked.